lang/test/org/partiql/lang/eval/ExprValueFactoryTest.kt

```diff
@@ -365,4 +369,86 @@ class ExprValueFactoryTest {
         Assert.assertEquals("Expected month to be 02", 2, timestamp.month)
         Assert.assertEquals("Expected day to be 29", 29, timestamp.day)
     }
+
+    private fun createIonTimeStruct(hour: Int, minute: Int, second: Int, nano: Int = 0, totalSeconds: Int? = null): IonStruct =
+        ion.newEmptyStruct().apply {
+            add("hour", ion.newInt(hour))
+            add("minute", ion.newInt(minute))
+            add("second", ion.newFloat( second + nano * 10.0.pow(-9)))
+            add("timezone_hour", ion.newInt(totalSeconds?.div(3600)))
+            add("timezone_minute", ion.newInt(totalSeconds?.div((60))?.rem(60)))
+            addTypeAnnotation("\$partiql_time")
+        }
```

Comment:
This looks like copy/paste to me...

Reply:
I have removed the `ionValue` assertion from here as those assertions are covered in `EvaluatingCompilerDateTimeTests`.